packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -140,6 +142,25 @@ class VectorGraphic extends StatefulWidget {
   State<VectorGraphic> createState() => _VectorGraphicWidgetState();
 }
 
+/// Load the [PictureInfo] from a given [loader].
+///
+/// It is the caller's responsibility to handle disposing the picture when
+/// they are done with it.
+Future<PictureInfo> loadPicture(
```

Comment:
Should we call this something else?  It's a bare top level method, it seems a little ambiguous.

We could make a vg utility class or something

Reply:
I could add a new method to the loader interface instead perhaps?

Reply:
or call it `LoadPictureForCustomerMoney`

Reply:
In flutter_svg there's a class called `Svg` with a private ctor and `public final Svg svg = Svg._()` for utility methods like this.

That way they don't polute the namespace of whatever imports them and you don't need weird imports like `import 'package:vector_graphics/vector_graphics.dart as vg` and `import package:vector_graphics/vector_graphics.dart show VectorGraphic` or whatever

Reply:
impl looks fine to me though

Reply:
https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

Reply:
They're not static members.

```dart
class VectorGraphicUtilities {
  VectorGraphicUtilities._();

  Future<Picture> loadPicture(...) {}
}

final VectorGraphicUtilities vg = VectorGraphicUtilities._();
```

Reply:
Technically not against the rules lol.

Reply:
will do

Reply:
done